cashu/core/crypto/dlc.py

```diff
@@ -0,0 +1,100 @@
+from hashlib import sha256
+from typing import List, Optional, Tuple
+
+from secp256k1 import PrivateKey, PublicKey
+
+
+def sorted_merkle_hash(left: bytes, right: bytes) -> bytes:
+    '''Sorts `left` and `right` in non-ascending order and
+        computes the hash of their concatenation
+    '''
+    if left < right:
```

Comment:
this should be changed to `if right < left`

Reply:
Forgot to fix it.